credsweeper/app.py

```diff
@@ -303,7 +306,9 @@ def data_scan(self, data_provider: DataContentProvider, depth: int, recursive_li
                                 f"{file_path}: size {zfl.file_size} is over limit {recursive_limit_size} depth:{depth}")
                             continue
                         with zf.open(zfl) as f:
-                            zip_content_provider = DataContentProvider(data=f.read(), file_path=file_path)
+                            zip_content_provider = DataContentProvider(data=f.read(),
+                                                                       file_path=file_path,
+                                                                       info=f"{data_provider.info}|ZIP")
```

Comment:
It seems `info` only used in case of zip files..(ZIP, GZIP, RAW)
I think it is not that useful feature that we should add new parameter `info`.